processing/CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.0] - 2021-04-30
+## [1.0.0] - 2021-05-04
 
 ### Added
 * Initial implementation of the module.
 * Add ST_VORONOIPOLYGONS function based on buffer (turfjs v6.3.0).
 * Add ST_VORONOILINES function based on buffer (turfjs v6.3.0).
+* Add ST_DELAUNAYPOLYGONS function.
```

Comment:
Here, maybe @Jesus89  can give us advice on what to do. Do we go for 1.1.0 2021-05-04 or put everything together?

Reply:
Yes. I have changed the base branch to merge this one in Voronoi's PR. But the rule is: releases on the same day will be included in the same version number (in this case 1.0.0).

Reply:
Ok, let's wait